pykern/util.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""utils for pykern
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern import pkconfig
+from pykern import pkinspect
+from pykern import pkio
+from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
+import os.path
+import sys
+
+
+_DEFAULT_ROOT = "run"
+
+
+def cfg_db_dir(db_root_env_name):
+    """Generate pkconfig parser function with errors for db's env var name
+
+    Args:
+        db_root_env_name (str): env var name for run dir root
+
+    Returns:
+        function: parser function for pkconfig.init
+    """
+    _NOT_ABS_ERROR = "{}: " + f"{db_root_env_name} must be absolute"
+    _NO_DIR_ERROR = "{}: " + f"{db_root_env_name} must be a directory and exist"
+
+    def cfg_db_dir_parser(v):
+        """Config value or root package's parent or cwd with `_DEFAULT_ROOT`"""
```

Comment:
The value shouldn't need to be printed either. Rather, this should be handled by the https://github.com/radiasoft/pykern/issues/74